shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -170,7 +170,15 @@ private void copyPath(File src, AlluxioURI dstPath) throws IOException {
           closer.close();
         }
       } else {
-        mFileSystem.createDirectory(dstPath);
+        // Add a judgement, because the dstPath maybe already exists. For example:
+        // 'copyFromLocal pathToSrcDir /'.
+        try {
+          mFileSystem.createDirectory(dstPath);
+        } catch (FileAlreadyExistsException e) {
```

Comment:
@yupeng9  I think it is common to copy a directory from local to existing directory. And even you run `bin/alluxio copyFromLocal srcDir /`, it only return you `/` . not 'the file already exits' .